src/hooks/use-document-sync-to-firebase.ts

```diff
@@ -77,6 +77,22 @@ export function useDocumentSyncToFirebase(
     }
   });
 
+    // sync properties for published documents
+    useSyncMstNodeToFirebase({
+      firebase, model: document.properties, path: `${metadata}/properties`,
+      enabled: readOnly && (user.id === uid) &&
+        [ProblemPublication, PersonalPublication, LearningLogPublication, SupportPublication ].includes(type),
```

Comment:
I don't see how this can possibly work for multi-class supports. It's synchronizing to firebase, but multi-class supports aren't _in_ firebase.

Reply:
Still don't see it but at least not we are not writing extra records into firebase